### ISSUE TITLE
Equipment changes while in a exosuit nullspaces your equipment

### ISSUE THREAD

**Issue by DoctoreMad** [tgstation/tgstation#28519]
Step 1. Get in a exosuit with some gear
Step 1.5. Put a normal guy with the same gear by your exosuit side for comparison
Step 2. Do something like using a Colossus's Strange Crystal to change someone into a clown(This is what happened on the server) or activating anything else that drops your equipment properly.
3.Watch as the person inside the Exosuit loses all his equipment instead of having it drop on the ground

**Comment by GunHog** [comment]
Are you sure it is nullspace and not going inside of the mech itself? The proc might transfer items to the mob's loc, in which case it would be the mech.

**Comment by DoctoreMad** [comment]
Well I asked a admin to test it but he didn't submit a issue, maybe I should have asked
How do you get items out of the mech though?

**Comment by DoctoreMad** [comment]
If you mean it just drops on the tile is on the exosuit you can guess again

**Comment by GunHog** [comment]
No, I mean _inside_ the mech. Not the turf. For example, get in a mech and use the admin `spawn` verb to spawn an object. Then, VV (view variables) the mech and look at the `contents` list. Your spawned object will be inside.

**Comment by XDTM** [comment]
Someone used loc instead of get_turf. Then again, there should be a way to let clothes fall inside a locker but not inside a mech. Or to give mechs emptiable storage.

**Comment by GunHog** [comment]
Ripley has a cargo bay.

**Comment by DoctoreMad** [comment]
well, if I knew how to retrieve items form the contents list, I would be able to check that

**Comment by XDTM** [comment]
keep the issue, mech contents are not accessible so the items are pretty much lost.

**Comment by DoctoreMad** [comment]
Ah fuck it a coder set a proper title
